Mathlib/Topology/Algebra/InfiniteSum/IntNat.lean

```diff
@@ -0,0 +1,337 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl
+-/
+
+import Mathlib.Topology.Algebra.InfiniteSum.Group
+import Mathlib.Logic.Encodable.Lattice
+
+/-!
+# Infinite sums over `ℕ` and `ℤ`
```

Comment:
Nitpick: this title first lists `Nat` and then `Int`. Maybe the filename should also be `NatInt.lean`?